websites/W/Wolvesville/presence.ts

```diff
@@ -285,49 +566,133 @@ presence.on("UpdateData", async () => {
 						if (playerCountPreGame === 1)
 							presenceData.state = `${playerCountPreGame} player connected`;
 						else presenceData.state = `${playerCountPreGame} players connected`;
+
+						presenceData.smallImageText = `${root.getAttribute(
+							"premid-username"
+						)} - Waiting for the game to begin`;
 					}
 				} else {
 					//Playing
-					presenceData.details = "In a game";
+					let gamemode: string,
+						playerState: string,
+						gameState = "",
+						gameOver = false,
+						rankedLeague = "unknown",
+						lang = "unknown";
+
+					if (!privacyMode) {
+						const gamemodeId = localStorage?.getItem("last-game-game-mode");
+						if (gamemodeId?.includes("sandbox")) gamemode = "Sandbox";
```

Comment:
@Slowlife01 @EncryptedDev we allow this or nah

Reply:
Nah

Reply:
A few presences do this if there is no other way of obtaining this data